processes/security_team_offboarding.md

```diff
@@ -0,0 +1,23 @@
+# Off-boarding Existing Security WG Members
+
+This document aims to provide a clear process for off-boarding existing WG members due to inactivity or any other eligible reason.
+
+## Process
+
+Every 6 months the WG will run through a process of validating membership relevancy as follows:
+
+* An issue will be posted on the WG Repository that will mention all existing WG members to query whether participation in the WG is still relevant.
+* After 2 weeks a reminder notice will be posted
+* After 4 weeks those who haven't responded to the issue will be placed in a pending status ('emeritus')
```

Comment:
emeritus is not really a pending status, instead a remaining recognition of the contributions made once they have been removed.

Reply:
thanks for clarifying. do you think the following is clearer?
```
* After 2 weeks a reminder notice will be posted
* After 8 weeks those who haven't responded to the issue will be removed from the WG (see `Revoking Access to Confidential Systems` section for details on access removal check-list) and be recognized as emeritus members.
```

I thought it makes sense that we will just move members who haven't responded to an emeritus recognition list instead of moving them after 4 weeks, then deleting them from that list when they chime in in the last 8 weeks period.

Reply:
@mhdawson it means that effectively the grace time is really 8 weeks, so if we have to restore someone's access after removing everything isn't it just extra work? It will probably be a rare case anyway so I'll update the texts as you suggested with clarifying the 4 weeks middle period

Reply:
SGTM